### PR TITLE
Add unpublishFeedGen script

### PR DESCRIPTION
**Pull request by dolciss**
This PR is based on a feature request from #76,
and I am modifying my own script to applying it in the main repository.

### CODE REVIEW

File: scripts/unpublishFeedGen.ts
Comment (ebwinters) [inline scripts/unpublishFeedGen.ts:15]: Can we read this from input prompt or .env? Seems safer than people potentially hardcoding password and pushing it

**Comment by ebwinters** [comment]
Hey just wondering, who reviews PRs in this repo? Have had one outstanding for a while and would like to get it merged or rejected

File: scripts/unpublishFeedGen.ts
Comment (dolciss) [reply scripts/unpublishFeedGen.ts:15]: Thanks for the review!
Yes, I am aware of the possibility of hard coding and pushing too.
However, I am doing the same once the PR #50 for publishFeedGen has not been merged.
Once PR #50 is merged, we will fix this as well :)

**Review by haileyok — APPROVED** [review]
thank you!

File: scripts/unpublishFeedGen.ts
Comment (haileyok) [reply scripts/unpublishFeedGen.ts:15]: I know I'm super late to the party 😭

I added inquirer to get all of these values, since it is just too darn easy to make a commit right now with a credential and, frankly, its a bit easier to just run the script real quick and enter inputs then have to make changes to the file itself. If anyone wants to tweak it still though, feel free.

**Pull request merged**
